sqlglot/optimizer/rewrite_subqueries.py

```diff
@@ -7,6 +7,7 @@
 def rewrite_subqueries(expression):
     expression = expression.copy()
     expression = decorrelate_subqueries(expression)
+    expression = expand_multi_table_selects(expression)
```

Comment:
Are these subqueries? Maybe we should have two different steps:

"decorrelate_subqueries"
"expand_multi_table_selects"

Instead of both in "rewrite_subqueries"?